docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/ApplyCustomWordEmbedding.cs

```diff
@@ -0,0 +1,80 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ApplyCustomWordEmbedding
```

Comment:
>ApplyCustomWordEmbedding [](start = 24, length = 24)

is this sample used anywhere ?

Reply:
Look at the overloaded version of `ApplyWordEmbedding` method in TextCatalog. It is referenced as an example there similar to other methods.

---
In reply to: [270986934](https://github.com/dotnet/machinelearning/pull/3142#discussion_r270986934) [](ancestors = 270986934)